_posts/2014-10-26-student-robotics-2015-starts-with-a-range-of-new-kit.md

```diff
@@ -18,10 +18,10 @@ The game: Capture the Flag
 
 <img class="right" src="{{ site.baseurl }}/images/content/arena-2015.png" title="The 2015 arena layout" alt="An image of the 2015 arena layout">
 
-This year's competition is all about retrieving flags from around the arena and putting them into the competitor's zone 
+This year's competition is all about retrieving flags from around the arena and putting them into the competitor's zone
 to get points. The flags are built from wooden cubes on ball casters.
 
-Read more about the [new game](/schools/game) or for full details please reference the [rulebook](/resources/2015/rulebook.pdf).
+Read more about the [new game](https://en.wikipedia.org/wiki/Student_Robotics#2015:_.22Capture_the_Flag.22) or for full details please reference the rulebook (now removed)
```

Comment:
The rulebooks from previous years are not currently hosted anywhere. However, I think that we should probably add them back in either in this repo, or more likely the docs repo. They provide an interesting insight into the history of the games.